back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/idea_plaintext_parser_service.rb

```diff
@@ -2,24 +2,16 @@
 
 module BulkImportIdeas
   class IdeaPlaintextParserService
-    QUESTION_TYPES = %w[select multiselect text text_multiloc multiline_text html_multiloc]
     TEXT_FIELD_TYPES = %w[text text_multiloc]
     MULTILINE_FIELD_TYPES = %w[multiline_text html_multiloc]
 
     FORBIDDEN_HTML_TAGS_REGEX = %r{</?(div|p|span|ul|ol|li|em|img|a){1}[^>]*/?>}
     EMPTY_SELECT_CIRCLES = ['O', '○']
     EMPTY_MULTISELECT_SQUARES = ['☐']
 
-    def initialize(project_id, locale, phase_id)
-      @project = Project.find(project_id)
-      @phase = phase_id ? @project.phases.find(phase_id) : TimelineService.new.current_phase(@project)
-      @custom_fields = IdeaCustomFieldsService.new(
-        Factory.instance.participation_method_for(@phase || @project).custom_form
-      )
-        .enabled_fields
-        .select { |field| QUESTION_TYPES.include? field.input_type }
-
-      @locale = locale || @locale
+    def initialize(custom_fields, locale)
```

Comment:
Simplified by passing in the custom fields as we are using the same set elsewhere in the import.